src/Migrations/Version20241205103739.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+declare(strict_types=1);
+
+namespace BigGive\Identity\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+final class Version20241205103739 extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return 'Delete unwanted account';
+    }
+
+    public function up(Schema $schema): void
+    {
+        $this->addSql(
+            "DELETE FROM Person WHERE stripe_customer_id = 'cus_RKaL5ZzHdUoftB' LIMIT 1"
```

Comment:
reviewer, please double-check the stripe id. Don't put any other details of the person on this public PR.